dashboard/config/programming_expressions/gamelab/camera-on.json

```diff
@@ -27,5 +27,5 @@
   "return_value": "No return value. Possibly changes output in the display after drawSprites() is called.",
   "short_description": "Activates the camera. The display area will be drawn according to the camera position and scale until camera.off() is called.",
   "syntax": "camera.on();",
-  "tips": "- The camera is automatically turned on at the start of each `draw()` loop. "
+  "tips": "- The camera is automatically turned on at the start of each [`function draw() {}()`(#fff176)](/docs/gamelab/draw/) loop. "
```

Comment:
`function draw() {}()` looks weird, might be an extra case of extra parens

Reply:
That is how the name looks, which is why this changed. There is a bigger question here (which relates to your comment below) about whether we should keep the existing text and just link it instead of replacing with the name. I'd love @tess323 's opinion here.

I think this would be a quick fix fwiw

Reply:
i think `function draw() {}` would be fine, but here it is `function draw() {}()` (note extra parens at the end)

Reply:
there are some places where `draw` got replaced and doesn't have those extra parens, so I think `draw()` got replaced with extra ones

Reply:
Good catch! I was having trouble with the double parentheses -- I fixed some but not all (and these would be hard to track down). I'm going to try changing this to just use what's already in the text but with the links, which might prevent this issue.